it/v3/endpoints/CreateAndAmendEmploymentExpensesControllerISpec.scala

```diff
@@ -29,14 +29,13 @@ import shared.support.IntegrationBaseSpec
 class CreateAndAmendEmploymentExpensesControllerISpec extends IntegrationBaseSpec {
 
   "Calling the amend endpoint" should {
-    "return a 200 status code" when {
+    "return a 204 status code" when {
       "any valid request is made" in new NonTysTest {
         override def setupStubs(): Unit =
           DownstreamStub.onSuccess(DownstreamStub.PUT, downstreamUri, NO_CONTENT, JsObject.empty)
 
         val response: WSResponse = await(request().put(requestBodyJson))
-        response.status shouldBe OK
-        response.json shouldBe hateoasResponse
```

Comment:
this can be replaced to verify that there is no body
`response.body.isEmpty shouldBe true`